core/chains/evm/gas/models.go

```diff
@@ -78,8 +78,8 @@ func NewEstimator(lggr logger.Logger, ethClient evmclient.Client, cfg Config, ge
 		return NewWrappedEvmEstimator(NewBlockHistoryEstimator(lggr, ethClient, cfg, geCfg, bh, *ethClient.ConfiguredChainID()), df, l1Oracle)
 	case "FixedPrice":
 		return NewWrappedEvmEstimator(NewFixedPriceEstimator(geCfg, bh, lggr), df, l1Oracle)
-	case "Optimism2", "L2Suggested":
-		return NewWrappedEvmEstimator(NewL2SuggestedPriceEstimator(lggr, ethClient), df, l1Oracle)
+	case "Optimism2", "L2Suggested", "SuggestedPrice":
```

Comment:
Optimism2 was deprecated some time ago. Since this is a cleanup PR, can we remove it from here if it doesn't affect anything else?

Reply:
Sounds good

Reply:
Done.